lib/migrate/Migrator.js

```diff
@@ -43,7 +43,10 @@ class Migrator {
       this.knex.userParams = this.knex.userParams || {};
     }
 
-    this.config = getMergedConfig(this.knex.client.config.migrations);
+    this.config = getMergedConfig(
+      this.knex.client.config.migrations,
+      this.knex.client.logger
```

Comment:
The second argument here needs to be `undefined`, not `this.knex.client.logger`

Reply:
Sorry for not replying before, but yeah, that was a keen observation, thank you for it.